src/backend/web/routes/index.js

```diff
@@ -7,4 +7,6 @@ router.get('/', (req, res) => {
   res.sendFile(path.join(__dirname, '../../../frontend/index.html'));
 });
 
+router.use('/admin', require('./admin'));
```

Comment:
let's keep all the imports in one location at the top of the file.

Reply:
I do not believe this can go above line 4 where we declare router.

Reply:
that's okay, after router its fine. Move the require there, store it in a variable and replace the on this line with said variable

Reply:
I think that for the purpose of using Router and modularize our routes, doing `require` on other Router is a good practice since we will knnow exactly which file is associated to which parent route.

Reply:
@vitokhangnguyen  The idea is to keep all declarations at the top of the code so they are easy to find. Additionally, you can provide a useful and meaningful name to the import which will be more descriptive than the name of the file. Storing the require in a variable also allows us to change that name later on.  Also, image that your file is located in a folder structure such as:
```/foo/bar/baz/finally.js```

instead of writing ```router.use('/some/route/', require('./foo/bar/baz/finally.js')``` we can write
```router.use('/some/route/', finally)``` which is significantly cleaner.

I'd be happy to read any documentation stating otherwise, but I've yet to find an example in the [Express Router](https://expressjs.com/en/guide/routing.html) documentation and definitely not what is being stated in the [NodeJS best practices repository](https://github.com/goldbergyoni/nodebestpractices#-38-require-modules-first-not-inside-functions).

Reply:
I think that it is easier to find the file if it is imported near where it is used, especially when it is guaranteed to be used only once (the file is small and a Route file does not usually has more than 1 export). Besides, I do not find what meaning a variable name can transmit that a filename cannot. In fact, having extra variables in the code file make it harder to trace the code.

I say this comes down to coding style and the author has his or her owned. I prefer to see what Router linked to what path so I can just jump there when I need to inspect the code, rather than having to trace the variable around the code (I might not know that I can find it at the top) and then the path.

Reply:
Again, the idea is to make this codebase easy to maintain. Additionally, your export could be an object with many methods, given your preference, I'd have to require and call the method all in the same line. You could say that we can make an exception in this case and put it at the top since the import is an object but then you'd be mixing 2 styles and making things more confusing for someone later on. Importing at the top is standard practice. It's what was taught in web222 and web322 and good practice as I've shown in the linked documentation above. You may be right it comes down to style, but we are aiming for consistency here and it's not about preference. This is the main reason we have a linter. We are not coding for ourselves but for the maintainers and future contributors to the codebase long after we are gone. If you fail to see how variable names can provide meaning and clarity to your code, then maybe read [this](https://medium.com/mindorks/meaningful-names-a-dimension-of-writing-clean-code-fdae1ae4f0b1) article.  I also dont want to have a file named some like: someRidiculouslyLongFileNameJustBecauseIDontWantToUseMeaningfulVariableNames.js. I will happily change my mind if you can link to credible documentation stating importing at the top is not both good and standard practice.

Reply:
Let's just get this code in and clean up later